.github/workflows/publish-to-pypi.yml

```diff
@@ -1,6 +1,10 @@
 name: Publish Python 🐍 distributions 📦 to PyPI
 
-on: workflow_dispatch
```

Comment:
thanks for changing, left this in whilst trying to experiment with pypi publishing last night